### ISSUE TITLE
Error when updating composer

### ISSUE THREAD

**Issue by martin-josef** [cedaro/satispress#164]
Hello, I've been using satispress (successfully!) for quite some time and it has really helped my way of working with bedrock. I now run the repository locally as I don't need it to be exposed to the internet. 

I get the following error: 

```
Warning: Accessing satis.test over http which is an insecure protocol.
http://satis.test/satispress could not be fully loaded ("http://satis.test/satispress/packages.json" does not contain valid JSON
Parse error on line 1:
<br /><b>Fatal erro
^
Expected one of: 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '['), package information was loaded from the local cache and may be out of date
```

I have reset (wp db reset & site empty) the satispress WordPress install, it hasn't changed. 

Kind regards, 
Martin

**Comment by raphaelparent** [comment]
I also get this when I try to run an update, the difference being I'm running it publicly over https

```
https://packages.raphparent.website/satispress could not be fully loaded ("https://packages.raphparent.website/satispress/packages.json" does not contain valid JSON
Parse error on line 1:
object(SatisPress\HT
^
Expected one of: 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '['), package information was loaded from the local cache and may be out of date
```

**Comment by raphaelparent** [comment]
@martin-josef reverting back to 1.0.0 seems to fix the issue on my end.

**Comment by bradyvercher** [comment]
As far as I can tell, the original issue is because [Composer requires HTTPS by default](https://getcomposer.org/doc/06-config.md#secure-http).

The other issues should be solved in #155 and #165.
